### ISSUE TITLE
Be more explicit about the limitations of R5 detailed itineraries in the doc

### ISSUE THREAD

**Issue by FlxPo** [r5py/r5py#386]
TLDR:
- Use detailed itineraries with caution, some are way off.
- Suggestion: be more explicit about R5 limitations in the doc ?

I am currently developing a python package that tries to make the mobility behavior of local populations as easy as possible. We need to compute travel times and distances for different transport modes for this, and I thought I could use r5py to estimate them.

It turns out it works great for travel times with `TravelTimeMatrixComputer`. But distances and times computed with `DetailedItinerariesComputer` are sometimes really off, because the routing algorithm seem to not choose the fastest path (at least for cars).

The developer of R5 says this is to be expected and that they won't try to fix it, in this [issue](https://github.com/conveyal/r5/issues/863) :

> We haven't used the point-to-point routing server for many years and it is not actively maintained. This dates from a time when we were exploring use of R5 as a replacement for OpenTripPlanner, but there has since been a clearer division of roles between the two projects with OpenTripPlanner 2 handling passenger-facing and point-to-point routing, while R5 is specialized in many-to-many urban analytics use cases. The point-to-point routing code paths do work somewhat differently than the one-to-many routing we use heavily for travel time mapping and accessibility calculations. We have not (yet) removed the point-to-point functionality in case anyone finds it useful, but realistically we probably won't spend time exploring issues with this part of the codebase.

I would suggest to be more explicit about what r5py cannot really do early in the documentation (which is great otherwise). In my case, I read : multi modal travel time matrices, cool, detailed itineraries, great, I can get travel times, distances and detailed route geometries !

r5py turned out not to fit our needs, but thanks anyway, this package is great.
